docs/user-guide/get-started.md

```diff
@@ -29,18 +29,6 @@ npx stylelint "**/*.css"
 
 _You should include quotation marks around file globs._
 
-If you use a pretty printer alongside Stylelint, you should turn off any conflicting rules. For example, you can use [Prettier's shared config](https://www.npmjs.com/package/stylelint-config-prettier) to do that:
```

Comment:
[Note] We don't need to suggest using the config anymore as the configs won't conflict.